std_msgs/Header header

float32 f_x
float32 f_y
float32 f_z

float32 f_azimuth_angle
float32 f_azimuth_angle_std

uint8 u_invalid_flags

float32 f_elevation_angle
float32 f_elevation_angle_std
float32 f_range
float32 f_range_std

float32 f_range_rate
float32 f_range_rate_std
int8 s_rcs
uint16 u_measurement_id
uint8 u_positive_predictive_value
uint8 u_classification
uint8 u_multi_target_probability
uint16 u_object_id
uint8 u_ambiguity_flag
uint16 u_sort_index